convert2rhel/systeminfo.py

```diff
@@ -24,6 +24,9 @@
 import time
 
 from collections import namedtuple
+from datetime import datetime as dt
+
+import rpm
```

Comment:
```suggestion
```

Reply:
not being used

Reply:
also the `from datetime import datetime as dt` is not being used

Reply:
good catch